index.html

```diff
@@ -3121,6 +3127,14 @@ <h3 id="applying">
             prefix.
           </p>
           <p>
+            We encourage implementors to add proprietary extensions to our
+            <a href=
+            "https://github.com/w3c/manifest/wiki/Extensions-Registry">Extensions
+            Registry</a>. This allows the community to track what extension to
+            be made and periodically consider those extensions for
+            standardization.
```

Comment:
the idea of extensions usually is that (a) they are "standards" in the sense that they must be documented and this documentation should be linked from the registry. making sure that there (a) is documentation, and (b) what has been documented does not violate the processing model of the foundation spec is something that often is handled by some sort of feedback/review when registrations are made. whether the documentation is or is intended to become a standard in itself is not so important, as long as it is documented and follows the extension model.